src/youbora-adapter.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * @license
+ * Youbora Plugin JS Kaltura player
+ * Copyright NicePopleAtWork & Kaltura
+ * @author Jordi Aguilar & Dan Ziv
+ */
+import $YB from './youbora.lib.min'
+import {VERSION, PLAYER_NAME} from 'playkit-js'
+import * as pkg from '../package.json'
+
+$YB.plugins.KalturaV3 = function (player, options) {
+  try {
+    /** Name and platform of the plugin.*/
+    this.pluginName = PLAYER_NAME;
```

Comment:
is part of the payload. player property
![image](https://user-images.githubusercontent.com/17685520/29364222-746bdeda-829b-11e7-903d-4c2ca619d7b0.png)
****